README.md

```diff
@@ -40,7 +36,15 @@ Markdown format:
 - [Talks](#talks)
 
 ## Computer Vision
+
 ### Image Representation Learning
+
+#### 2009
+
+- Learning Long-Range Vision for Autonomous Off-Road Driving
```

Comment:
Is it better to move this to Robotic section?

Reply:
Yes, you're right. I will move it there.

Reply:
Fixed.